addons/account/views/report_invoice.xml

```diff
@@ -82,6 +85,14 @@
                         <td class="text-right">
                             <span t-esc="', '.join(map(lambda x: (x.description or x.name), l.invoice_line_tax_ids))"/>
                         </td>
+                        <td t-if="display_taxamount" class="text-right">
+                            <t t-set="taxes" t-value="l.invoice_line_tax_ids.compute_all(l.price_unit, l.currency_id, l.quantity)['taxes']"/>
+                            <t t-set="tax_amount" t-value="0.0"/>
+                            <t t-foreach="taxes" t-as="tax">
+                                <t t-set="tax_amount" t-value="tax_amount + tax['amount']"/>
+                            </t>
+                            <span t-esc="tax_amount" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/>
+                        </td>
```

Comment:
why not:
` <t t-set="tax_amount" t-value="l.invoice_line_tax_ids.compute_all(l.price_unit, l.currency_id, l.quantity)['total_included']"/>`
It should be the same result.